index.html

```diff
@@ -146,8 +146,9 @@ <h2>Scope</h2>
                 however, it may emit warnings if it encounters deprecated
                 features.</li>
               <li>Facilitate mapping to HTML5/CSS3.</li>
-              <li>Should address the <a href="https://www.w3.org/TR/media-accessibility-reqs/">Media
+              <li>Address the <a href="https://www.w3.org/TR/media-accessibility-reqs/">Media
                   Accessibility User Requirements</a>.</li>
+              <li>Facilitate contribution of real time streaming.</li>
```

Comment:
@tairt concern about potential misunderstanding or confusion with e.g. DASH. Request rewording to narrow scope to contribution between authoring and distribution encoders for example. See summary from EBU Tech3370.

Reply:
@tairt I think I've addressed this - please re-review.